python/pyspark/broadcast.py

```diff
@@ -29,8 +29,10 @@
 
 if sys.version < '3':
     import cPickle as pickle
+    protocol = 2
 else:
     import pickle
+    protocol = 3
```

Comment:
guess we can use pickle.DEFAULT_PROTOCOL and pickle.DEFAULT_PROTOCOL - 1 just a thought not sure it matters

Reply:
The same code exists in serializers.py, I have simply tried to maintain consistency.